src/syntax/Parser.js

```diff
@@ -2715,7 +2715,7 @@ export class Parser {
    *
    * @return {ParseTree}
    */
-  parseExpression_(expressionIn = Expression.IN) {
+  parseExpression_(expressionIn = Expression.NORMAL) {
```

Comment:
I have a follow up task to replace this with a simple boolean parameter (with a constant for readability)